### PR TITLE
6002 bugfix pie charts

### PR DESCRIPTION
**Pull request by emitche**
- Fix numbers in pie charts
- When tree is loaded and expanded, pie chart displays as expected
- On window resize, resize charts
- If a breakeven question is expanded initially before user interaction, display the pie charts

### CODE REVIEW

**Comment by emitche** [comment]
I'm really happy with this approach. Thanks, @hooverlunch.

File: app/assets/javascripts/backbone/views/loan_charts_view.coffee [hunk 1]
```diff
@@ -16,6 +16,15 @@ class MS.Views.LoanChartsView extends Backbone.View
       },
       backgroundColor: 'none'
     }
+    @loadGoogleCharts
+
+    # When window is resized, resize the charts
+    window.addEventListener("resize", @loadGoogleCharts.bind @);
```
Comment (hooverlunch) [inline app/assets/javascripts/backbone/views/loan_charts_view.coffee]: Single quotes for JS please.

File: app/assets/javascripts/backbone/views/loan_questionnaires_view.coffee [hunk 2]
```diff
@@ -14,6 +14,11 @@ class MS.Views.LoanQuestionnairesView extends Backbone.View
     'confirm:complete .linked-document [data-action="delete"]': 'removeLinkedDocument'
     'click .edit-action': 'editDocument'
     'focus .questionnaire-form form': 'setupDirtyForm'
+    'tree.open': 'notifyExpandListeners'
+
+  # Add a custom event for tree expansion. This event is listened to by LoanChartsView.
+  notifyExpandListeners: (e) ->
+    @$(e.node.element).find('[data-tree-expand-listener]').trigger('tree.expanded')
```
Comment (hooverlunch) [inline app/assets/javascripts/backbone/views/loan_questionnaires_view.coffee]: Nice. I'm happy with this solution!

File: app/assets/javascripts/backbone/views/loan_questionnaires_view.coffee [hunk 3]
```diff
@@ -78,6 +83,7 @@ class MS.Views.LoanQuestionnairesView extends Backbone.View
     nodes
 
   editDocument: (e) ->
+    # Fire a global event. Consider refactoring this later to use the style used in LoanChartsView
```
Comment (hooverlunch) [inline app/assets/javascripts/backbone/views/loan_questionnaires_view.coffee]: Perhaps change this to read `the style used above in notifyExpandListeners`.

**Pull request merged**